recipes/msgpack-cxx/all/conanfile.py

```diff
@@ -15,6 +15,15 @@ class MsgpackCXXConan(ConanFile):
     license = "BSL-1.0"
     no_copy_source = True
 
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "use_boost": [True, False],
+    }
+    default_options = {
+        "use_boost": True
```

Comment:
```suggestion
        "use_boost": True,
```

you know that you can aggregate reviews in only one commit? It avoids to forget one.

Reply:
well, it's too minor thing to invalidate existing reviews. more like a style preference, as it doesn't provide any actual advantage for the package consumer.